src/DurableTask.Netherite/Connections/ConnectionInfo.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace DurableTask.Netherite
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Storage;
+
+    /// <summary>
+    /// Internal abstraction used for capturing connection information and credentials.
```

Comment:
Considering this type is `public`, should "Internal" be removed from the comment?

Reply:
yes